### PR TITLE
Feature/ob 32 layout static page

### PR DESCRIPTION
**Pull request by MillanaCh**
Static layout page in src-> app-> services -> beard
I did also responsive version for other devices.

### CODE REVIEW

File: src/app/services/beard/page.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        bg-cover
+        bg-[position:center_bottom,_left_bottom_0.3rem]
+        after:bg-[url('/images/bg/white-grunge.svg')]
+        after:bg-repeat-x
+        after:w-full
+        after:h-20
+        after:absolute
+        after:bg-[length:400px]
+        after:bottom-[10]
+
```
Comment (ain-odin) [inline src/app/services/beard/page.tsx:26]: Perhaps, this is only my problem, but this part is not displayed on the page
Screenshot 2023-09-13 at 13 12 52

Comment (ain-odin) [inline src/app/services/beard/page.tsx:37]: Maybe this is only my problem, but this part isn't displayed on the page
Screenshot 2023-09-13 at 13 12 52

Comment (Alex-2kZharkov) [reply src/app/services/beard/page.tsx:37]: Strange, I see it okay:

image

**Comment by Alex-2kZharkov** [comment]
@MillanaCh you need to add the Footer component to this page

File: src/app/services/beard/page.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
+'use client';
+
+import Image from 'next/image';
+import { Header } from '../../../components/Header';
+
+export default function Beard() {
```
Comment (Alex-2kZharkov) [inline src/app/services/beard/page.tsx]: return type: `ReactElement`

File: src/app/services/beard/page.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  const defaultTextClassName = 'leading-8 my-5 text-lg';
+  const defaultImageClassName = 'my-8 max-h-[32rem] object-cover';
+  return (
```
Comment (Alex-2kZharkov) [inline src/app/services/beard/page.tsx:20]: empty line before the `return`

File: src/app/services/beard/page.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    <main>
+      <div
```
Comment (Alex-2kZharkov) [inline src/app/services/beard/page.tsx]: use the `main` right here, no need to create additional div wrapper

File: src/app/services/beard/page.tsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+        after:w-full
+        after:h-20
+        after:absolute
+        after:bg-[length:400px]
+        after:bottom-[10]
+        
+        "
+      >
+        <div className="px-12">
+          <Header />
```
Comment (Alex-2kZharkov) [inline src/app/services/beard/page.tsx]: the `<main>` tag cannot contain `<header>` tag. Header, then main, then footer

**Review by Alex-2kZharkov — CHANGES_REQUESTED** [review]

**Review by Alex-2kZharkov — APPROVED** [review]
Good!

**Review by Alex-2kZharkov — APPROVED** [review]

**Pull request merged**